centreon/www/front_src/src/Dashboards/Dashboards.tsx

```diff
@@ -1,13 +1,147 @@
-import { useTranslation } from 'react-i18next';
-
-import { Typography } from '@mui/material';
-
-import { labelDashboards } from './translatedLabels';
-
-const Dashboards = (): JSX.Element => {
-  const { t } = useTranslation();
-
-  return <Typography>{t(labelDashboards)}</Typography>;
-};
-
-export default Dashboards;
+import { Suspense, useMemo } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { useAtom, useAtomValue, useSetAtom } from 'jotai';
+
+import { TiledListingPage } from '@centreon/ui';
+import {
+  DashboardForm,
+  DashboardFormLabels,
+  DashboardResource,
+  Header,
+  Modal
+} from '@centreon/ui/components';
+
+import {
+  labelCancel,
+  labelCreate,
+  labelCreateDashboard,
+  labelDashboards,
+  labelDelete,
+  labelDeleteDashboard,
+  labelDescription,
+  labelDescriptionDeleteDashboard,
+  labelName,
+  labelUpdate,
+  labelUpdateDashboard
+} from './translatedLabels';
+import ListingSkeleton from './Skeleton';
+import {
+  closeDialogAtom,
+  deleteDialogStateAtom,
+  isDialogOpenAtom,
+  selectedDashboardAtom
+} from './atoms';
+import useSubmitDashboard from './useSubmitDashboard';
+import Listing from './Listing';
+import useRemoveDashboard from './useRemoveDashboard';
+
+import { ModalActionsLabels } from 'packages/ui/src/components/Modal/ModalActions';
+
+const Dashboards = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const [deleteDialogState, setDeleteDialogState] = useAtom(
+    deleteDialogStateAtom
+  );
+  const isDialogOpen = useAtomValue(isDialogOpenAtom);
+  const selectedDashboard = useAtomValue(selectedDashboardAtom);
+  const closeDialog = useSetAtom(closeDialogAtom);
+
+  const { submit } = useSubmitDashboard();
+  const { remove: removeDashboard } = useRemoveDashboard();
+
+  const labels = useMemo(
+    (): {
+      deleteConfirmation: { actions: ModalActionsLabels; description: string };
+      form: DashboardFormLabels;
+      modalTitle: { create: string; delete: string; update: string };
+    } => ({
+      deleteConfirmation: {
+        actions: {
+          cancel: t(labelCancel),
+          confirm: t(labelDelete)
+        },
+        description: t(labelDescriptionDeleteDashboard)
+      },
+      form: {
+        actions: {
+          cancel: t(labelCancel),
+          submit: {
+            create: t(labelCreate),
+            update: t(labelUpdate)
+          }
+        },
+        entity: {
+          description: t(labelDescription),
+          name: t(labelName)
+        }
+      },
+      modalTitle: {
+        create: t(labelCreateDashboard),
+        delete: t(labelDeleteDashboard),
+        update: t(labelUpdateDashboard)
+      }
+    }),
+    [t]
```

Comment:
```suggestion
    []
```
useMemo will shallow compare the function. But the function reference will never be the same throughout renders thus it's useless to add the `t` in the dependency list. Either remove, the `t` or remove the useMemo